pallets/remote-asset-manager/src/tests.rs

```diff
@@ -0,0 +1,2 @@
+// Copyright 2021 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
Are there any tests for the XCM test utils that show it works basically?

Reply:
have a look at the tests of the simulator example
https://github.com/shaunxw/xcm-simulator/blob/master/example/src/main.rs#L115
basically it sets up a connection between relay and para so that messages can be exchanged. So you would send a message from para -> relay and then check that this message was executed as expected on relay.

Reply:
@clearloop I'm not too sure about this direct import solutions. this is necessary because in relay.rs we depend the XcmRouter that is generated in the decl_test_network! macro.
I guess it will become more obvious once we add xcm tests to other pallets.

Reply:
I'm feeling okay with the 

```
#[path = "..."]
mod xx;
```

solution if it works, we need the test util, but don't have to develop it, soon there will be a better solution